### PR TITLE
import sequence_* API to new namespace

### PR DESCRIPTION
**Pull request by iclementine**
### PR types

Others

### PR changes

APIs

### Describe

1. import `sequence_mask` into `paddle.nn.functional', since it is a commonly used API in sequence modelding tasks;
2. import `sequence_*` APIs (except for sequence_mask) into `paddle.static.nn`.

### CODE REVIEW

File: python/paddle/nn/functional/__init__.py [hunk 1]
```diff
@@ -97,6 +97,7 @@
 # from .extension import temporal_shift  #DEFINE_ALIAS
 # from .extension import warpctc  #DEFINE_ALIAS
 from .extension import diag_embed  #DEFINE_ALIAS
+from .extension import sequence_mask  #DEFINE_ALIASs
```
Comment (willthefrog) [inline python/paddle/nn/functional/__init__.py]: `#DEFINE_ALIAS` is no longer needed, also, a typo (`s`)

**Review by willthefrog — CHANGES_REQUESTED** [review]

**Review by willthefrog — COMMENTED** [review]
LGTM,  @jzhang533  @XiaoguangHu01

File: python/paddle/nn/functional/__init__.py (hunk 1 above)
Comment (iclementine) [reply python/paddle/nn/functional/__init__.py]: Done

**Review by jzhang533 — COMMENTED** [review]
API搬家后，相应的docstring也麻烦更新一下。

**Review by jzhang533 — COMMENTED** [review]
sequence_mask是挪到`paddle.nn.functional`下的API，docstring里要统一下Tensor, LoDTensor, Variable这些概念吧。
其他的挪到`paddle.static`下的API感觉就那样吧。

**Comment by iclementine** [comment]
图片
图片

**Review by jzhang533 — APPROVED** [review]
lgtm

**Review by willthefrog — APPROVED** [review]

**Comment by iclementine** [comment]
图片

图片

**Pull request merged**